db/migrate/20210514135324_add_audits_audited_changes_index.rb

```diff
@@ -0,0 +1,7 @@
+class AddAuditsAuditedChangesIndex < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :audits, :audited_changes, using: :gin, algorithm: :concurrently
```

Comment:
will this be ok with our `statement_timeout`, I think it might take longer than 20 seconds?

Reply:
we mitigated this last time with: https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/db/migrate/20210319163050_add_auditable_type_id_index_to_audits.rb#L7

Reply:
makes, will do the same...